qemu/tests/cfg/win_virtio_single_driver.cfg

```diff
@@ -0,0 +1,50 @@
+-win_virtio_single_driver:
+    type = win_virtio_single_driver
```

Comment:
@suqinhuang Yes. you are right. 
but it is not planing in this patch,as this is only a sub-test cases
I would like to write a new patch for that.